client/app/dedicated/server/translations/Messages_fr_FR.xml

```diff
@@ -586,6 +586,7 @@
    <translation id="server_configuration_firewall_asa_disable_step1_question" qtlid="183905">Souhaitez-vous réellement désactiver le firewall Cisco ASA pour votre serveur <span class="normal italic">{0}</span>&#160;?</translation>
    <translation id="server_configuration_firewall_asa_disable_success" qtlid="151828">La demande de désactivation du firewall Cisco ASA a bien été prise en compte et sera effective dans quelques minutes.</translation>
    <translation id="server_configuration_firewall_asa_disable_fail" qtlid="151841">Une erreur est survenue lors de l'désactivation du firewall Cisco ASA</translation>
+   <translation id="server_configuration_service_expired" qtlid="191421">Le service est expiré.</translation>
```

Comment:
why do we display this tab is it's empty for expired services ? Maybe we could either disable it or remove it

Reply:
Yes, so in this case the whole master page and it's tabs should be disabled